tests/integration/test_workflow_db_mgr.py

```diff
@@ -106,9 +104,10 @@ async def test_db_upgrade_pre_803(
     schd: Scheduler = scheduler(reg, paused_start=True)
     with pytest.raises(sqlite3.OperationalError):
         async with start(schd):
-            assert (
-                ('n_restart', '1') in db_select(schd, False, 'workflow_params')
-            )
+            pass
+    assert (
+        ('n_restart', '1') in db_select(schd, False, 'workflow_params')
+    )
```

Comment:
I think this `assert` was not being hit due to the `OperationalError` raising before then

Reply:
I actually tested it - pretty sure it was being hit!

Reply:
Re-confirmed, it does get hit in the original.   But, your version is probably easier to understand (since it's not obvious exactly when the sqlite3 exception gets thrown).